docs/src/test/scala/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala

```diff
@@ -130,7 +130,7 @@ class DebuggingDirectivesExamplesSpec extends RoutingSpec {
     }
     def printResponseTime(log: LoggingAdapter) = {
       val requestTimestamp = System.nanoTime
-      akkaResponseTimeLoggingFunction(log, requestTimestamp)(_)
+      akkaResponseTimeLoggingFunction(log, requestTimestamp)_
```

Comment:
That's pretty obscure...

Reply:
Yes... I thought the same, luckily the error was telling what to do